Update hiring page with details for senior developer and product mana… (#7342)
opencollective/opencollective-frontend: components/static-pages/hiring.md

<strong>All position are currently filled, check back later! You can still send a spontaneous application if you wish.</strong>
-->

> <em>January 28th 2021</em>. We are currently hiring for the following roles:

Reviewer: ```suggestion
<em>January 12th 2022</em>. We are currently hiring for the following roles:
```

<ul>

>   <li><a href='#developer'>Senior, 'Full Stack' (Node) Developer</a></li>

Reviewer: ```suggestion
  <li><a href='#developer'>Senior, 'Full Stack' Software Engineer</a></li>
```

We usually don't use the word "Developer".

>   <li><a href='#developer'>Senior, 'Full Stack' Software Engineer</a></li>

Reviewer: ```suggestion
  <li><a href='#software-engineer'>Senior, 'Full Stack' Software Engineer</a></li>
```

Watch this 2 minute explainer video about what we do and why we are passionate about it:

<iframe src="https://www.youtube.com/embed/IBU5fSILAe8?wmode=opaque&amp;enablejsapi=1" scrolling="no" width="854" height="480" frameborder="0"></iframe>

Like the movements that we support, we value openness and transparency. We strive to create inclusive, respectful and sustainable communities and we're 100% Open Source. If you want to know more about us, you can already join our [Slack](https://slack.opencollective.com) and check out the [Open Collective Way](https://docs.opencollective.com/help/about/the-open-collective-way) to learn more about how we operate and what we stand for.

We are a fully remote, international team - with team members in New Zealand, Mexico, India, Brazil, Canada, the United Kingdom, Spain, Germany and France. We are an equal opportunities employer and welcome applications from anyone with the right mix of experience and skills.

> <h2 id='developer'>Senior, 'Full Stack' (Node) Developer</h2>

Reviewer: ```suggestion
<h2 id='software-engineer'>Senior, 'Full Stack' Software Engineer</h2>
```

We are looking for a senior full-stack engineer who’s excited about open source, making communities sustainable, and transparent finances.

### We think you’d be great at this if you:

- Are comfortable in a remote environment that is at the same time highly collaborative (we [apply](https://github.com/opencollective/opencollective/issues/2267) a [seeking feedback process](https://reinventingorganizationswiki.com/theory/decision-making/)) and autonomous (you can run a feature end to end on your own!)

> - Are based within 4 hours of Central European Time:

Reviewer: This excludes US east coast?

Author: ```suggestion
- Are based within 5 hours of Central European Time:
```

> - Are based within 5 hours of Central European Time:

Reviewer: 6 hours between CET and EST.

> - Are based within 5 hours of Central European Time:

Reviewer: ```suggestion
- Are based within 6 hours of Central European Time:
```

  - our GraphQL API (powered by Node, PostgreSQL and Sequelize)
  - our Web App (powered by Next.js, React and Apollo).

- You are excited about our mission and share our philosophy.

<h2 id='product-manager'>Senior Product Manager</h2>

We are looking for a senior product manager who is excited about creating a world in which it is as rewarding and financially secure to build and maintain software for the commons as it is for the corporation.

### We think you’d be great at this if you:

- Are comfortable in a remote environment that is at the same time highly collaborative (we [apply](https://github.com/opencollective/opencollective/issues/2267) a [seeking feedback process](https://reinventingorganizationswiki.com/theory/decision-making/)) and autonomous (you will own complete features and products!)

> - Are based within 4 hours of Central European time:

Reviewer: Same timezone than the other?

> - Are based within 4 hours of Central European time:

Reviewer: ```suggestion
- Are based within 6 hours of Central European time:
```
